lib/widgets/screens/sign_in/sign_in_screen.dart

```diff
@@ -64,161 +65,342 @@ class _SignInScreenState extends State<SignInScreen> {
-                                      scaffoldManager
-                                          .showSnackBar(const SnackBar(
-                                        content: Text(
-                                            'Please enter you correct password'),
-                                      ));
-                                      passwordController.text = '';
+                                    } else {
+                                      router.push('/');
                                     }
-                                  } else {
-                                    router.push('/');
                                   }
-                                }
-                              },
-                              child: Padding(
-                                padding: const EdgeInsets.all(8.0),
-                                child: Text(
-                                  'Sign In',
-                                  style: GoogleFonts.poppins(
-                                    fontSize: 14,
-                                    fontWeight: FontWeight.bold,
+                                },
+                                child: Padding(
+                                  padding: const EdgeInsets.symmetric(
+                                    vertical: Insets.widgetSmallInset,
+                                    horizontal: Insets.widgetLargeInset * 2.5,
+                                  ),
+                                  child: Text(
+                                    l10n.logIn,
+                                    style: TextStyles.largeTextButton(context),
                                   ),
                                 ),
                               ),
                             ),
+                          ],
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Insets.widgetSmallInset),
+                      child: TextButton(
+                        key: const Key('registerButton'),
+                        onPressed: () {
+                          _openSignUpScreen(context);
+                        },
+                        child: Text(
+                          l10n.createNewAccount,
+                          style: TextStyles.sectionSubHeaderBold(context),
+                        ),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.all(Insets.widgetSmallInset),
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: [
+                          Container(
+                            width: Dimensions.loginBoxWidth / 2,
+                            height: Dimensions.lineHeight,
+                            color: Colors.black,
+                          ),
```

Comment:
It might be easier to use a Divider widget. Also you can play with flex values to make the widgets occupy all of the available horizontal space, instead of using a constant value.

Reply:
Hey Miguel, 
The reason I didn't use the divider widget is because the divider widget takes the whole row. In this scenario, there are two horizontal lines separated with the word "or". I will try to play around with the divider to see it the rendering makes sense, and comment here.